src/test/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporterTest.java

```diff
@@ -349,7 +349,7 @@ private void assertMetric(String metricName, double metricValue, String realm, T
 
             final StringBuilder builder = new StringBuilder();
 
-            builder.append(metricName).append("{");
+            builder.append(metricName+"_total").append("{");
```

Comment:
as it's a stringbuilder, you could use append here too.
`builder.append(metricName).append("_total").append("{");`

it's slight more performatic

Reply:
Shure, it's up to maintainer. this is only a proposal which works, if you are maintainer, i could change it.

Reply:
thanks @ghenadiibatalski this is fine inside the unit test